packages/server/src/threads/query.js

```diff
@@ -33,10 +33,12 @@ class QueryRunner {
       return fields
     }
     const bindings = findHBSBlocks(sql)
-    let variables = []
+    let variables = [],
+      arrays = []
     for (let binding of bindings) {
-      let variable = integration.getBindingIdentifier()
-      variables.push(binding)
+      // look for array/list operations in the SQL statement, which will need handled later
+      const listRegex = new RegExp(`(in|IN|In|iN) ${binding}`)
```

Comment:
NAB: Might be worth doing `(in|IN|In|iN)\s+${binding}` here - as extra spaces are valid SQL as well.

Reply:
Good catch, didn't think of that!